package.json

```diff
@@ -50,13 +54,15 @@
     "express": "^4.18.2",
     "google-protobuf": "^3.18.0",
     "http-terminator": "^3.2.0",
+    "@microsoft/durabletask-js": "^0.1.0-alpha.1",
     "node-fetch": "^2.6.7"
   },
   "devDependencies": {
     "@types/body-parser": "^1.19.1",
     "@types/express": "^4.17.15",
     "@types/jest": "^27.0.1",
     "@types/node": "^16.9.1",
+    "@types/readline-sync": "^1.4.8",
```

Comment:
Is this required here? Or just examples?

Reply:
sorry removed it and ts-node which only used in examples.